app/services/management/auth.js

```diff
@@ -68,20 +68,23 @@ const login = async (referrerUri = null) => {
   setReferrerUri(referrerUri);
 
   /* setting promiseType = native does not work for later functions inside the closure
-    checkLoginIframe: true breaks for some users in chrome causing an infinite loop
-    see: https://szoradi-balazs.medium.com/keycloak-login-infinite-loop-9005bcd9a915
+  checkLoginIframe: true breaks for some users in chrome causing an infinite loop
+  see: https://szoradi-balazs.medium.com/keycloak-login-infinite-loop-9005bcd9a915
   */
-  await keycloak.init({
-    onLoad: 'login-required',
-    checkLoginIframe: false,
-  });
+  if (!keycloak.authenticated === undefined || !keycloak.authenticated) {
+    await keycloak.init({
+      onLoad: 'login-required',
+      checkLoginIframe: false,
+    });
+  } else {
+    await keycloak.updateToken(Number.MAX_SAFE_INTEGER);
+  }
 };
 
 const logout = async () => {
   try {
-    await keycloak.init();
```

Comment:
what was this line doing?

Reply:
this doesn't need to happen, since for logout to work, the `keycloak` object instance should already be initialized

Tested with this line gone, nothing out of the ordinary happened 

THOUGH, this can come back to bite me for some obscure situation